app/javascript/bundles/MainApp/components/RestaurantItem.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import StarRatings from 'react-star-ratings';
+
+/**
+ * Contains one restaurant item in the RestaurantsList
+ */
+export default class RestaurantItem extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  convertCuisine = (cuisine) => {
+    return String.fromCharCode((cuisine in RestaurantItem.cuisines) ? RestaurantItem.cuisines[cuisine] : 102);
```

Comment:
102  - what does that mean?
extracting to a const with a name can eliminate this "magic number"

Reply:
Done